<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPropCompareFolder</class>
 <widget class="QDialog" name="QPropCompareFolder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Folder compare</string>
  </property>
  <widget class="QLabel" name="IDC_STATIC_COMPARE_FOLDER">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>10</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhMultiLine</set>
   </property>
   <property name="text">
    <string>Compare method:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="IDC_COMPAREMETHODCOMBO">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>5</y>
     <width>141</width>
     <height>33</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="IDC_COMPARE_STOPFIRST">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>50</y>
     <width>404</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>S&amp;top after first difference</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="IDC_IGNORE_SMALLTIMEDIFF">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>74</y>
     <width>444</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ign&amp;ore time differences less than 3 seconds</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="IDC_COMPARE_WALKSUBDIRS">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>98</y>
     <width>404</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Include unique subfolders contents</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="IDC_RECURS_CHECK">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>122</y>
     <width>404</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Include Subfolders</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="IDC_EXPAND_SUBDIRS">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>146</y>
     <width>404</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Automatically expand all subfolders</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="IDC_IGNORE_REPARSEPOINTS">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>170</y>
     <width>404</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ignore &amp;Reparse Points</string>
   </property>
  </widget>
  <widget class="QLabel" name="IDC_STATIC_QUICK_COMPARE">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>200</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhMultiLine</set>
   </property>
   <property name="text">
    <string>Quick compare limit (MB):</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="IDC_COMPARE_QUICKC_LIMIT">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>224</y>
     <width>60</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="IDC_STATIC_BINARY_COMPARE">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>260</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhMultiLine</set>
   </property>
   <property name="text">
    <string>Binary compare limit (MB):</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="IDC_COMPARE_BINARYC_LIMIT">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>284</y>
     <width>60</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="IDC_STATIC_COMPARE_THREADS">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>308</y>
     <width>460</width>
     <height>60</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhMultiLine</set>
   </property>
   <property name="text">
    <string>Number of compare threads (a negative value implies addition of the number of available CPU cores):</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="IDC_COMPARE_THREAD_COUNT">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>360</y>
     <width>60</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_COMPAREFOLDER_DEFAULTS">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>416</y>
     <width>140</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Defaults</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
